openedx/features/student_account/helpers.py

```diff
@@ -38,7 +47,8 @@ def get_non_active_course(user):
         today = datetime.now(utc).date()
 
         try:
-            course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
+            # course = CourseOverview.objects.get(id=user_course.course_id, end__gte=today)
+            course = CourseOverview.objects.get(id=user_course.course_id)
```

Comment:
Would it work the same without adding the `end__gte` check?

Reply:
Also, please remove the commented out code.

Reply:
It was commented to test... That needs to be removed.